base/workflow/R/do_conversions.R

```diff
@@ -83,7 +83,8 @@ do_conversions <- function(settings, overwrite.met = FALSE, overwrite.fia = FALS
     # met conversion
     
     if (input.tag == "met") {
-      name <- ifelse(is.null(settings$browndog), "MET Process", "BrownDog")
+      # Previously it was checked if Browndog was used. Although, Browndog is being rempved from PEcAn now
```

Comment:
I think this is a case where we don't need a comment -- once the old code is gone, nobody will think to wonder about it.
```suggestion
```